master/manifests/flannel-migration/calico.yaml

```diff
@@ -0,0 +1,784 @@
+---
```

Comment:
These manifests have html in them.  Is there some directive missing?
https://deploy-preview-2817--calico.netlify.com/master/manifests/flannel-migration/calico.yaml